let edad;

try {
    console.log({nombre : nombre});
} catch (error){
    console.warn(error.messague)
} finally {
    console.log("se ha terminado la ejecusion")
}